src/System.Net.HttpListener/src/System/Net/HttpListenerPrefixCollection.cs

```diff
@@ -39,6 +39,11 @@ internal HttpListenerPrefixCollection(HttpListener listener)
         public void CopyTo(Array array, int offset)
         {
             _httpListener.CheckDisposed();
+            // Preserve compatability with full .NET Framework.
+            if (array == null)
+            {
+                throw new NullReferenceException();
```

Comment:
That said, I don't understand why this was necessary to add... the very next line accesses `array.Length`, so it will throw a NullReferenceException if `array` is null.